client/web-sveltekit/src/routes/search/StreamingProgress.svelte

```diff
@@ -25,23 +27,19 @@
         )
     }
 
-    $: severity = progress.skipped.some(skipped => skipped.severity === 'warn' || skipped.severity === 'error')
-        ? 'error'
-        : 'info'
     $: hasSkippedItems = progress.skipped.length > 0
     $: sortedItems = sortBySeverity(progress.skipped)
     $: openItems = sortedItems.map((_, index) => index === 0)
     $: suggestedItems = sortedItems.filter((skipped): skipped is Required<Skipped> => !!skipped.suggested)
     $: hasSuggestedItems = suggestedItems.length > 0
+    $: isError = state === 'error'
 </script>
 
 <Popover let:registerTrigger let:toggle placement="bottom-start">
-    <Button variant="secondary" size="sm" outline>
+    <Button variant={`${isError ? 'danger' : 'secondary'}`} size="sm" outline>
```

Comment:
This should work too:

```suggestion
    <Button variant="{isError ? 'danger' : 'secondary'}" size="sm" outline>
```